File: xml/narwhal/read/xml_read.js
#! /usr/local/bin/narwhal
// ------------------------------------------------------------------
//
//	xml_read.js
//
//						May/09/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/narwhal_common/text_manipulate.js');
var xml_manipulate = require ('/var/www/data_base/common/narwhal_common/xml_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");

var args = require('system').args;
var file_in = args[1];

var str_in = readFile(file_in);

var dict_aa =  xml_manipulate.xml_to_dict_proc (str_in);

text_manipulate.dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: xml/jython/read/xml_read.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	jython/read/xml_read.py
#
#						May/27/2015
#
# --------------------------------------------------------------------
import	sys
reload(sys)
sys.setdefaultencoding('utf-8')
sys.path.append ('/var/www/data_base/common/jython_common')
#
from jython_xml_manipulate import jython_xml_to_dict_proc
#
from jython_text_manipulate import dict_display_proc
from jython_file_io import file_to_str_proc
# --------------------------------------------------------------------
print ("*** 開始 ***")
#
xml_file="/var/tmp/xml_file/cities.xml"
#
xml_str=file_to_str_proc (xml_file)
dict_aa = jython_xml_to_dict_proc (xml_str)
dict_display_proc (dict_aa)
#
print ("*** 終了 ***")
# --------------------------------------------------------------------
